### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Lupie**
Proofread existing text
Reorganize sections logically
Edit for a broader audience
Add links
Update Matt's title

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -77,14 +81,12 @@ We have an open source quick start template for nodejs.
...
-registers with hyperbahn.
-
 You can `tcurl` it directly at `localhost:9000` or `tcurl` it through
-hyperbahn at `127.0.0.1:21300`.
+Hyperbahn at `127.0.0.1:21300`. If you're unfamiliar with `tcurl`, the
+command-line tool to call TChannel servers, learn about it [here][tcurl].
 
-Because the service is available on Hyperbahn anyone can talk to it
-if they can talk to Hyperbahn.
+Once a service is available on Hyperbahn, any other service on Hyperbahn
```
Comment (Lupie) [inline README.md]: Am I correct in thinking any other service on THAT Hyperbahn cluster can talk to it?
It's not like some little startup can set up a Hyperbahn cluster, put Service A on it, and then have Service A talk to services at Uber that are on our Hyperbahn network, correct?

File: README.md [hunk 2]
```diff
@@ -77,14 +81,12 @@ We have an open source quick start template for nodejs.
...
+can talk to it.
 
 ## Installation
```
Comment (Lupie) [inline README.md]: This seems misplaced. Would it help to add some context, or should we put it another section?

File: README.md (hunk 1 above)
Comment (mranney) [inline README.md]: Yes, there are many Hyperbahns, and they don't know about each other. You can run a cluster on your laptop, and it won't know about prod.

File: README.md [hunk 3]
```diff
@@ -1,72 +1,76 @@
-# hyperbahn
+# Hyperbahn
 
-Service discovery and routing solution.
+Service discovery and routing system for large-scale microservice operations.
```
Comment (mranney) [inline README.md]: I now favor losing the word "system", so:

Service discovery and routing for large-scale microservice operations

File: README.md [hunk 4]
```diff
@@ -1,72 +1,76 @@
...
- - circuit breaking to cut off broken clients
+ - **Configuration Discovery** - Stop managing host/port files or configurations.
+ - **Timeouts** - Transitively enforce services' SLAs. Every request
+   must specify a timeout, moving towards a fast failure model.
+ - **Retries** - Eliminate transient failures within a request timeout window.
+ - **Load Balancing** - Spread calls evenly across a service, and distribute outgoing
+   requests evenly over connected Hyperbahn nodes.
+ - **Rate Limiting** - Shield services from excessive calls.
+ - **Circuit Breaking** - Prevent cascading failure and cut off broken clients. Build
+   fast failure into all layers of the network.
```
Comment (mranney) [inline README.md]: I would also add:

**Distributed Tracing** - Understand the entire call flow using Zipkin.

**Pull request merged**